pkg/registry/apis/secret/decrypt/v0alpha1/decrypt.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package decrypt.v0alpha1;
+
+option go_package = "github.com/grafana/grafana/pkg/registry/apis/secret/decrypt/v0alpha1";
```

Comment:
```suggestion
option go_package = "github.com/grafana/grafana/pkg/apis/secret/v0alpha1/decrypt";
```
if the goal is to use a client outside this package... i would add the definitions somewhere that could eventually be imported. 

or similar

Reply:
Makes sense! I'll update that 👍